ansible/roles/opencontrail-gateway/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: VRouter log directory
+  file: path="/var/log/contrail" state=directory
+
+- name: VRouter agent configuration
+  template: src=contrail-vrouter-agent.conf.j2 dest=/etc/contrail/contrail-vrouter-agent.conf
+
+- name: Redhat-sytle interface configuration
+  include: redhat.yml
+  when: ansible_distribution == "Fedora"
+
+- name: Create vrouter agent service
+  copy: src=contrail-vrouter-agent.service dest=/etc/systemd/system
+  notify:
+    - reload systemd
+
+- name: Docker pull image
+  command: /usr/bin/docker pull opencontrail/vrouter-agent:2.20
```

Comment:
Some tasks use  `docker` binary and some commands use full path `/usr/bin/docker`. These should be consistent across the roles.